packages/vm/examples/run-blockchain.ts

```diff
@@ -31,7 +31,7 @@ async function main() {
   // Note that this optimization is a bit hacky and might
   // not be working in the future though. :-)
   if (validatePow) {
-    blockchain._ethash!.cacheDB = level('./.cachedb')
+    (blockchain.consensus as any)._ethash!.cacheDB = level('./.cachedb')
```

Comment:
just a nit, can you update this example and BlockchainTestsRunner.ts to do this `as EthashConsensus`? and ethash doesn't need the `!` anymore since it is no longer optionally undefined.